app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -1,39 +1,45 @@
 <%
   email_signup_link ||= false
   email_signup_link_text ||= t("govuk_component.subscription_links.email_signup_link_text", default: "Get email alerts")
+  email_signup_link_data_attributes ||= false
```

Comment:
This section at the top of the component is getting quite large now, and making it difficult to follow what's happening in the view. Can we move some of this logic out into a helper?

Reply:
Gave it a go but by the time I've passed everything I need out to the helper and back again I'm not sure it actually saves any lines of code.

Reply:
Can we do something like we do for the related navigation component and just pass through the local assigns?

```<% related_nav_helper = GovukPublishingComponents::Presenters::RelatedNavigationHelper.new(local_assigns) %>```

Reply:
@vanitabarrett ah good idea, I'd forgotten you can do that, thanks. Have moved the code into a helper.